Desktop/Print SCU/packages.config.merge

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I see this file is still included in the PR. If you are able to, please remove it. Otherwise I can do it as a post-PR action.